.github/workflows/push-test.yml

```diff
@@ -0,0 +1,38 @@
+# Check for every push
+
+name: Unit test
+
+on:
+  push:
```

Comment:
[q] does this construct mean that this will be run on both PRs and merges to master?

Reply:
yes